src/logic/wallets/getWeb3.ts

```diff
@@ -28,9 +28,13 @@ export const WALLET_PROVIDER = {
 
 // With some wallets from web3connect you have to use their provider instance only for signing
 // And our own one to fetch data
+const httpProviderOptions = {
+  keepAlive: true,
```

Comment:
keepAlive is true by default: 
https://github.com/ethereum/web3.js/blob/1.x/packages/web3-providers-http/src/index.js#L44